### ISSUE TITLE
no use of zeta parameter (Madgwick)

### ISSUE THREAD

**Issue by neroner** [kriswiner/MPU9250#227]
First of all great work, thanks!

I would like to notice that you define the zeta parameter in the main program but there is no use of zeta parameter in your Madgwick update filter. 

In a Magwick's paper entitled 'An efficient orientation filter for inertial and inertial/magnetic sensor arrays,' there is the version with the zeta parameter. You use another version which is available from x-io.co (again from Madgwick). 

I have implemented both of versions and I take different results. Can you inform me why you used this version and what is the difference?

Thank you in advance!

**Comment by kriswiner** [comment]
Didn;t notice any effect of using zeta. The filter is computationally
efficient and produces good results as it is implemented here.

**Comment by neroner** [comment]
True. I take very good results. I just informed you that there is no use of zeta parameter... 

Thank you for your prompt reply

**Comment by kriswiner** [comment]
Thanks for the info...
